js/views/navigation.js

```diff
@@ -46,6 +51,11 @@ define(function(require) {
 				this.$el.hide();
 				$('#app-navigation-toggle').css('background-image', 'none');
 			}
+		},
+		_adjustButtonWidth: function() {
+			var newMailButtonContainer = $('#mail-new-message-fixed');
+			var innerWidth = $('#app-navigation-accounts').width();
+			newMailButtonContainer.attr('style', 'width: ' + innerWidth + 'px !important');
```

Comment:
Okay, I've given a thought. Move the first li (the button container) above the ul.
![capture d ecran_2016-05-05_07-51-31](https://cloud.githubusercontent.com/assets/14975046/15036929/3782f15a-1296-11e6-960e-2ecd1fddbc7c.png)

Remove the unused div container of the button (not the essential part, but clean code are always welcome :p )
And apply the following css (do it in console if you want, should override the header's css).

```
#mail-new-message-fixed {
    z-index: 1;
    padding-bottom: 12px;
    background-color: rgba(255, 255, 255, .75);
    padding: 10px;
}
#mail_new_message {
    width: 100%;
    padding: 10px;
    padding-left: 34px;
    background-position: 10px center;
    text-align: left;
}
.mail-account-email:first-child {
    display: none;
}
#app-navigation {
    overflow: hidden;
}
#app-navigation > ul {
    padding-bottom: 80px; /* scroll hack because of #settings fixed */
}
```